### PR TITLE
adding kap, drone, and VS Code extension

### PR DESCRIPTION
**Pull request by lucaperret**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ Take a look at [awesome-micro](https://github.com/amio/awesome-micro)! 😌
 - [aframe-next-static](https://github.com/michaltakac/aframe-next-static) - Build WebVR apps with A-frame and React, export and serve them as static sites on ZEIT's Now.
 - [next-init](https://www.npmjs.com/package/next-init) - Simplest next.js generator.
 - [next.js-typescript-starter-kit](https://github.com/deptno/next.js-typescript-starter-kit) - next.js@5 + redux + typescript, simple configuration for SEO, analytics. you can use styled-jsx in tsx.
+- [kap-now](https://github.com/lucaperret/kap-now) - Deploy [Kap](https://github.com/wulkano/kap) capture with now.
+- [drone-now](https://github.com/lucaperret/drone-now) - Deploying to now within [Drone CI](https://drone.io) pipeline.
```
Comment (matheuss) [inline readme.md]: `now` -> `Now`

File: readme.md [hunk 2]
```diff
@@ -57,6 +57,8 @@ Take a look at [awesome-micro](https://github.com/amio/awesome-micro)! 😌
 - [aframe-next-static](https://github.com/michaltakac/aframe-next-static) - Build WebVR apps with A-frame and React, export and serve them as static sites on ZEIT's Now.
 - [next-init](https://www.npmjs.com/package/next-init) - Simplest next.js generator.
 - [next.js-typescript-starter-kit](https://github.com/deptno/next.js-typescript-starter-kit) - next.js@5 + redux + typescript, simple configuration for SEO, analytics. you can use styled-jsx in tsx.
+- [kap-now](https://github.com/lucaperret/kap-now) - Deploy [Kap](https://github.com/wulkano/kap) capture with now.
```
Comment (matheuss) [inline readme.md]: I don't think this classifies as a boilerplate

Comment (lucaperret) [reply readme.md]: In the services part ?

Comment (timothyis) [reply readme.md]: I'd say this belongs in the showcase section, personally

**Comment by lucaperret** [comment]
Hi there, I've commit changes, do want to put these utility in another section ?

**Comment by timothyis** [comment]
@lucaperret I still believe these belong in the showcase section

**Comment by lucaperret** [comment]
@timothyis I've moved `kap-now` and `drone-now` to the showcase section.
I've also added the VS code extension.

**Pull request merged**